src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseApprovalService.cs

```diff
@@ -140,16 +141,22 @@ public class ReleaseApprovalService : IReleaseApprovalService
                     var userReleaseRoles =
                         await _userReleaseRoleService.ListUserReleaseRolesByPublication(ReleaseRole.Approver,
                             release.Publication.Id);
-
-                    var notifyReleaseApprovers = request.ApprovalStatus == ReleaseApprovalStatus.HigherLevelReview &&
-                                                 userReleaseRoles.Any();
                     
+                    
+                    var userPublicationRoles = await _context.UserPublicationRoles
+                        .Include(upr => upr.User)
+                        .Where(upr => upr.PublicationId == release.PublicationId && (upr.Role == PublicationRole.Owner || upr.Role == PublicationRole.ReleaseApprover))
+                        .ToListAsync();
+                    
+
+                    var notifyHigherReviewers = request.ApprovalStatus == ReleaseApprovalStatus.HigherLevelReview &&
+                        userReleaseRoles.Any() || userPublicationRoles.Any();
```

Comment:
You could move this closer to where it's referenced now, i.e. after the line `case ReleaseApprovalStatus.HigherLevelReview:`